### PR TITLE
Fixdata

### PR DESCRIPTION
**Pull request by rachelslau**

### CODE REVIEW

File: client/src/components/Organizer/DataEntry/DataEntry.jsx [hunk 1]
```diff
@@ -10,39 +10,49 @@ class DataEntry extends Component {
...
           waveNum: '',
-          fileName: 'UPLOAD FILE'
+          tablesName: 'UPLOAD FILE',
+          projectsName: 'UPLOAD FILE'
         })
       } else {
           this.setState({
             tableNum: result[0].tables,
-            clusterNum: result[0].clusters,
+            MaxNum: result[0].clusters,
```
Comment (lajiang2) [inline client/src/components/Organizer/DataEntry/DataEntry.jsx]: MaxNum should be maxNum

File: client/src/components/Organizer/DataEntry/DataEntry.jsx [hunk 2]
```diff
@@ -69,75 +79,101 @@ class DataEntry extends Component {
...
+    }
+    this.setState({
+      projectsName: fileName
     })
   }
 
   async postData() {
-    let results;
+    let tablesName = this.state.tablesName;
+    if (this.state.tablesName === "UPLOAD FILE") {
```
Comment (lajiang2) [inline client/src/components/Organizer/DataEntry/DataEntry.jsx]: single quotes same and below

File: server/index.js [hunk 3]
```diff
@@ -47,6 +47,7 @@ app.post('/api/dummy', async (req, res) => {
 app.put('/api/score/:judgeName', async (req, res) => {
   const { judgeName } = req.params;
   const { projectId, score } = req.body;
+  console.log(judgeName, projectId, score);
```
Comment (lajiang2) [inline server/index.js]: do we need this log?

File: server/index.js [hunk 4]
```diff
@@ -149,27 +125,19 @@ app.get('/api/data', async (req, res) => {
 })
 
 app.put('/api/data', async (req, res) => {
-  const { tables, clusters, waves} = req.body;
-  db.query('UPDATE dataentry SET tables = $1, clusters = $2, waves = $3, filename = $4;', [
+  console.log('entered put');
```
Comment (lajiang2) [inline server/index.js]: remove this console.log

File: server/index.js [hunk 5]
```diff
@@ -149,27 +125,19 @@ app.get('/api/data', async (req, res) => {
...
-  res.json("You successfully posted to dataentry");
-  });
-
-app.put('/api/csv', async (req, res) => {
-  const {filename, csv} = req.body;
   let i;
   for (i = 1; i < csv.length; i++) {
-    console.log(csv[i]); // to check that theyve all been put in
-
+    //console.log(csv[i]); // to check that theyve all been put in
```
Comment (lajiang2) [inline server/index.js]: delete this

**Review by lajiang2 — CHANGES_REQUESTED** [review]

**Review by lajiang2 — APPROVED** [review]

**Review by partyshah — APPROVED** [review]

**Pull request merged**